lib/ui/address_entry.dart

```diff
@@ -12,6 +12,7 @@ import 'package:wallet/wallet.dart';
 class AddressEntry extends StatelessWidget {
   final _controller = TextEditingController();
   final Function(bool)? onAddressChanged;
+  final Function(int)? onAmountChanged;
```

Comment:
Why is this not in `AmountEntry`?

Reply:
the call to launch scanner is in the address_entry https://github.com/Foundation-Devices/envoy/blob/dab57a4aaf0afa70a284aefb7e9ae4e947aca82c/lib/ui/address_entry.dart#L110-L118